src/main/java/org/gradle/profiler/ScenarioDefinition.java

```diff
@@ -83,4 +97,13 @@ public void visitProblems(InvocationSettings settings, Consumer<String> reporter
 
     protected void printDetail(PrintStream out) {
     }
+
+    protected String getExecutableName() {
+        return getBuildToolDisplayName().toLowerCase();
```

Comment:
This won't work on `Windows` since it needs the `.bat` extension, right?

Reply:
Why? IIUC, when we say `mvn`, Windows will automatically search for `mvn.exe`/`mvn.bat`/`mvn.cmd`, right? BTW, default `mvn` command is `mvn.cmd`.

Reply:
Does this work for `ProcessBuilder` as well or only from the command-line? Can you try this out on Windows?

Reply:
Seems like we need the extensions. I update the PR to `mvn.cmd`/`buck.exe`/`bazel.exe`